src/sections/Section1.jsx

```diff
@@ -27,61 +27,72 @@ const Animation = styled.div`
-            lottie.loadAnimation({
-                container: document.querySelector(`#derp-animation`),
-                animationData: derpAnimation,
-            });
-        }
-        //eslint-disable-next-line react-hooks/exhaustive-deps
-        animationLoaded = true;
-    }, []);
-
-    return (
-        <SSection
-            id={'Section1'}
-            gradient
-            center
-        >
-            <ImageContainer >
-                <HoprBall/>
-                {/*<Image*/}
-                {/*    alt="Hopr logo"*/}
-                {/*    src={'/images/derp_top-header.png'}*/}
-                {/*/>*/}
-                <Animation id='derp-animation' />
-            </ImageContainer>
+const Disclaimer = styled(Subtext)`
+  margin-top: 32px;
+  font-size: 14px;
+  text-align: center;
+`;
 
-            <Typography type="h2">
-                DERP<br/>
-                DUMB ETHEREUM RPC PROVIDER
-            </Typography>
-
-            <Subtext center>
-                Add the DERP RPC endpoint to your crypto wallet to see exactly what information is being leaked about you every time you connect to a crypto service.
-            </Subtext>
+function Section1(props) {
+  let animationLoaded = false;
+  useEffect(() => {
+    // check to prevent double animation load on page remount
+    if (!animationLoaded) {
+      lottie.loadAnimation({
+        container: document.querySelector(`#derp-animation`),
+        animationData: derpAnimation,
+      });
+    }
+    //eslint-disable-next-line react-hooks/exhaustive-deps
+    animationLoaded = true;
+  }, []);
 
-            <Button
-                hopr
-                onClick={props.setShowSetup}
-            >
-                SETUP
-            </Button>
+  return (
+    <SSection id={"Section1"} gradient center>
+      <ImageContainer>
+        <HoprBall />
+        {/*<Image*/}
+        {/*    alt="Hopr logo"*/}
+        {/*    src={'/images/derp_top-header.png'}*/}
+        {/*/>*/}
+        <Animation id="derp-animation" />
+      </ImageContainer>
 
+      <Typography type="h2">
+        DERP
+        <br />
+        DUMB ETHEREUM RPC PROVIDER
+      </Typography>
 
-        </SSection>
-    );
+      <Subtext center>
+        Add the DERP RPC endpoint to your crypto wallet to see exactly what
+        information is being leaked about you every time you connect to a crypto
+        service.
+      </Subtext>
+      <Button hopr onClick={props.setShowSetup}>
+        SETUP
+      </Button>
+      <Disclaimer>
+        Disclaimer: DERP is only an educational tool to showcase all the data
+        your wallet leaks about you. It does not solve the issue! If you want a
+        truly secure and private RPC provider, use{" "}
+        <a
+          href="https://hoprnet.org/rpch"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          RPCh.
+        </a>
+      </Disclaimer>
+    </SSection>
+  );
 }
 
 export default Section1;
```

Comment:
Not part of the task, why is it here?

Reply:
that's weird, don't know why this commit went here :/